blazor/treegrid/state-management.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: State management in Blazor TreeGrid Component | Syncfusion
+description: Learn here all about State management in Syncfusion Blazor TreeGrid component and more.
+platform: Blazor
+control: Tree Grid
+documentation: ug
+---
+
+# State management in Blazor TreeGrid Component
+
+Tree Grid Component retains its state using local storage on browser reload. Also it allows users to save and load tree grid state manually. The tree grid will use user-provided state to render instead of its properties provided declaratively.
+
+Below properties can be saved and loaded into tree grid.
+
+Property|
+-----|
+Columns |
+TreeColumnIndex |
+TreeGridFilterSettings |
+TreeGridSortSettings |
+TreeGridPageSettings
+
+## Enabling persistence in tree grid
+
+State persistence allows the tree grid to retains the current tree grid state in the browser local storage for state maintenance. This action is handled through the `EnablePersistence` property which disabled by default. When it is enabled, some properties of the tree grid will be retained even after refreshing the page.
```

Comment:
State persistence allows the tree grid to retain the current tree grid state in the browser's local storage for state maintenance. This action is handled through the `EnablePersistence` property, which is disabled by default. When it is enabled, some properties of the tree grid will be retained even after refreshing the page.

Reply:
made the changes